$NetBSD: patch-ab,v 1.3 2016/08/11 00:15:53 mef Exp $

(the first half)
date: 2006-04-07 11:23:19 +0900;  author: minskim;  state: Exp;
Use teTeX/module.mk to run mktexlsr for texmf-local only.

(second half)
date: 2011-07-09 23:19:30 +0900;  author: bsiegert;  state: Exp;  lines: +18 -3;
Add user-destdir support. Almost trivial except for one bug inthe install
script.

--- config/install.pl.orig	2002-11-29 11:49:01.000000000 +0000
+++ config/install.pl
@@ -372,12 +372,6 @@ foreach $item (sort keys %Install_items)
 
 if($cfg{TEXPATH}) {
   print "\nNote: trying to install LaTeX2HTML style files in TeX directory tree\n     ($cfg{TEXPATH})\n";
-  unless(mkpath($cfg{TEXPATH})) {
-  #my $testpath = $cfg{TEXPATH}; # to strip (latex2)html
-  #$testpath =~ s/[$dd$dd][^$dd$dd]*$//;
-  #if((-d $cfg{TEXPATH} && !-w _) || (-d $testpath && !-w _)) {
-    print STDERR "\nError: Cannot install LaTeX2HTML style files in $cfg{TEXPATH}\n";
-  } else {
     my $dir = 'texinputs';
     my $dest = $cfg{TEXPATH};
     unless(opendir(DIR,$dir)) {
@@ -394,12 +388,12 @@ if($cfg{TEXPATH}) {
         }
       }
       closedir(DIR);
-      if($ok && $cfg{MKTEXLSR}) {
-        print "Info: Running $cfg{MKTEXLSR} to rebuild ls-R database...\n";
-        system($cfg{MKTEXLSR});
-      }
+# print/teTeX/module.mk will run mktexlsr automatically.
+#      if($ok && $cfg{MKTEXLSR}) {
+#        print "Info: Running $cfg{MKTEXLSR} to rebuild ls-R database...\n";
+#        system($cfg{MKTEXLSR});
+#      }
     }
-  }
 }
 print "Done. Have a lot of fun with LaTeX2HTML!\n";
 exit 0;
